BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/file_collection/views/_plain_file_list_row.html.erb

```diff
@@ -36,20 +36,28 @@
 
 <td class="left">
     <%= ("&nbsp;" * 6 * file.depth).html_safe %>
-    <% if file.symbolic_type == :directory %>
-      <%= image_tag "/images/folder_icon_solid.png" %>
-    <% else %>
-      <%= image_tag "/images/file_icon.png" %>
-    <% end %>
-    <% if file.size > 0 %>
-      <%= data_link file.name, @userfile %>
-    <% else %>
-      <%= Pathname.new(file.name).basename.to_s %>
-    <% end %>
-    <% if file.symbolic_type == :directory %>
-      <span class="ajax_onclick_show_child small warning">Expand</span>
-      <span class="ajax_onclick_hide_child small warning" style="display:none">Hide</span>
-    <% end %>
+
+    <div style="display: inline-flex">
+      <% if file.symbolic_type == :directory %>
+        <%= image_tag "/images/folder_icon_solid.png" %>
+      <% else %>
+        <%= image_tag "/images/file_icon.png" %>
+      <% end %>
+
+      <%= "&nbsp".html_safe %>
+
+      <% if file.size > 0 %>
+        <%= data_link file.name, @userfile, file.size %>
+      <% else %>
+        <%= Pathname.new(file.name).basename.to_s %>
+      <% end %>
+
+      <% if file.symbolic_type == :directory %>
+        <%= "&nbsp".html_safe %>
```

Comment:
What ? First you're missing the semicolon, and also, why not simply directly `&nbsp;` instead of `<%= "&nbsp;".html_safe %>` ?!? This file is a HTML template!